backend/Squirrel.ConsoleApp/Controllers/SettingController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models.Models;
+
+namespace Squirrel.ConsoleApp.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class SettingController: ControllerBase
+{
+    private readonly IConnectionFileService _connectionFileService;
+
+    public SettingController(IConnectionFileService connectionFileService)
+    {
+        _connectionFileService = connectionFileService;
+    }
+    
+    [HttpPost]
+    [Route("connect")]
+    public IActionResult Post(ConnectionString connectionString)
+    {
+        _connectionFileService.SaveToFile(connectionString);
+
+        //Connection unique ID
+        var randomId = Guid.NewGuid();
```

Comment:
i am not sure this logic should be in controller

Reply:
What logic? If you about guid, so this is temporary solution

Reply:
then you can mark it with a comment like 'temporary solution, will create id in the future' or add 'TODO'